src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -611,6 +611,7 @@ const StyledMenu = styled('div')`
       .menu}; /* This is needed to be able to cover e.g. pagination buttons, but also be below dropdown actor button's zindex */
   right: 0;
   box-shadow: ${p => p.theme.dropShadowLight};
+  overflow: hidden;
```

Comment:
👀 Will this affect any tooltips? Maybe project selector?

Reply:
I couldn't find any tooltips on autocomplete items. The EventGroupingInfo, and AssigneeSelector components have a tooltip on the actor. ProjectSelector has a tooltip on the 'Add Project' button when the user doesn't have project:write but all of these tooltips display correctly.